lib/src/utils/kratos_message.dart

```diff
@@ -1,178 +1,1561 @@
+        return const ErrorValidationSettingsFlowExpired();
+
+      // Recovery Validation Error (4060000-4060099)
+      case ErrorValidationRecovery.id: // 4060000
+        return const ErrorValidationRecovery();
+      case ErrorValidationRecoveryRetrySuccess.id: // 4060001
+        return const ErrorValidationRecoveryRetrySuccess();
+      case ErrorValidationRecoveryStateFailure.id: // 4060002
+        return const ErrorValidationRecoveryStateFailure();
+      case ErrorValidationRecoveryMissingRecoveryToken.id: // 4060003
+        return const ErrorValidationRecoveryMissingRecoveryToken();
+      case ErrorValidationRecoveryTokenInvalidOrAlreadyUsed.id: // 4060004
+        return const ErrorValidationRecoveryTokenInvalidOrAlreadyUsed();
+      case ErrorValidationRecoveryFlowExpired.id: // 4060005
+        return const ErrorValidationRecoveryFlowExpired();
+      case ErrorValidationRecoveryCodeInvalidOrAlreadyUsed.id: // 4060006
+        return const ErrorValidationRecoveryCodeInvalidOrAlreadyUsed();
+
+      // Verification Validation Error (4070000-4070099)
+      case ErrorValidationVerification.id: // 4070000
+        return const ErrorValidationVerification();
+      case ErrorValidationVerificationTokenInvalidOrAlreadyUsed.id: // 4070001
+        return const ErrorValidationVerificationTokenInvalidOrAlreadyUsed();
+      case ErrorValidationVerificationRetrySuccess.id: // 4070002
+        return const ErrorValidationVerificationRetrySuccess();
+      case ErrorValidationVerificationStateFailure.id: // 4070003
+        return const ErrorValidationVerificationStateFailure();
+      case ErrorValidationVerificationMissingVerificationToken.id: // 4070004
+        return const ErrorValidationVerificationMissingVerificationToken();
+      case ErrorValidationVerificationFlowExpired.id: // 4070005
+        return const ErrorValidationVerificationFlowExpired();
+      case ErrorValidationVerificationCodeInvalidOrAlreadyUsed.id: // 4070006
+        return const ErrorValidationVerificationCodeInvalidOrAlreadyUsed();
+
+      // System Error (5000000-5000099)
+      case ErrorSystem.id: // 5000000
+        return const ErrorSystem();
+      case ErrorSystemGeneric.id: // 5000001
+        return const ErrorSystemGeneric();
+
+      default:
+        return const ErrorSystemGeneric();
+    }
+  }
+}
+
+// === Login related messages ===
+final class InfoSelfServiceLoginRoot extends KratosMessage {
+  const InfoSelfServiceLoginRoot();
+
+  static const id = 1010000;
+}
+
+final class InfoSelfServiceLogin extends KratosMessage {
+  const InfoSelfServiceLogin();
+
+  static const id = 1010001;
+}
+
+final class InfoSelfServiceLoginWith extends KratosMessage {
+  const InfoSelfServiceLoginWith._({
+    required this.provider,
+  });
+
+  static InfoSelfServiceLoginWith? maybeFromParameters(
+    ContextParameters? contextParameters,
+  ) {
+    if (contextParameters?['provider'] case final provider?) {
```

Comment:
```suggestion
    if (contextParameters case {'provider': final provider?}) {
```
The same for all map checks in this file